src/index.js

```diff
@@ -19,3 +21,4 @@ export {Tooltip} from "./Tooltip";
 export {Label} from "./Label";
 export {FileInput} from "./FileInput/FileInput";
 export * from "./InfoPanel";
+export {Number};
```

Comment:
it's a quirk of the es6 import/export pattern. `Number` is currently a default export, so it has to be imported as a default here and then exported as a secondary. (the others are exported as secondary imports, but i'm toying with the idea of providing default exports so clients can import individual components without pulling all of the components in)